### DISCUSSION TITLE
Where to download dev builds?

### DISCUSSION THREAD

**Discussion by MystPi** [ScratchAddons/ScratchAddons#5017] [Help] [answered]
Sorry if this has an obvious answer, but where can you download dev builds of SA? Every so often I've seen them mentioned in issues/pr's, so I was wondering where to try them out.

**Comment by apple502j** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/33279053/185793490-15c6a90b-0e37-4994-99ee-c6933b61c562.png)

**Reply by MystPi** [reply]
So you just download the ZIP of the repo?

**Reply by Samq64** [reply]
Or use git which is a best option if you want to try out different branches and keep it up-to-date.

**Reply by MystPi** [reply]
Okay, thanks you guys. 😸
